Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM maven:slim as builder
+
+LABEL maintainer="Juho Inkinen <juho.inkinen@helsinki.fi>"
+
+
+COPY src/ src/
+COPY pom.xml .
+RUN mvn package
+
+
+FROM tomcat:8-jdk11-slim
+
+RUN apt-get update \
+	&& apt-get install -y --no-install-recommends \
+		libvoikko1 \
+		voikko-fi \
+		gosu \
+	&& rm -rf /var/lib/apt/lists/* /usr/include/*
+
+COPY --from=builder /target/mauiserver-*-SNAPSHOT.war /usr/local/tomcat/webapps/mauiserver.war
```

Comment:
I would consider downloading the artifact from maven central. This works great for creating images from snapshots, but downloading a released war from maven central would allow for more repeatable an authorative builds.